code/modules/mob/living/simple_animal/bot/secbot.dm

```diff
@@ -276,21 +276,18 @@
 	playsound(src, 'sound/weapons/cablecuff.ogg', 30, TRUE, -2)
 	current_target.visible_message(span_danger("[src] is trying to put zipties on [current_target]!"),\
 						span_userdanger("[src] is trying to put zipties on you!"))
-	addtimer(CALLBACK(src, .proc/handcuff_target, target), 6 SECONDS)
+	addtimer(CALLBACK(src, .proc/handcuff_target, current_target), 6 SECONDS)
 
 /mob/living/simple_animal/bot/secbot/proc/handcuff_target(mob/living/carbon/current_target)
-	if(!(bot_mode_flags & BOT_MODE_ON)) //if he's in a closet or not adjacent, we cancel cuffing.
-		return
-	if(!isturf(current_target.loc))
-		return
-	if(!Adjacent(current_target))
+	if((!(bot_mode_flags & BOT_MODE_ON)) || !isturf(current_target.loc) || !Adjacent(current_target) || current_target.handcuffed  ) //if he's in a closet or not adjacent, we cancel cuffing.
```

Comment:
i split this into 3 separate checks for readability

Reply:
> i split this into 3 separate checks for readability

ok